lib/services/android-project-service.ts

```diff
@@ -126,10 +139,28 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 			}
 
 			this.cleanResValues(targetSdkVersion, frameworkVersion).wait();
-
+			if(this.canUseStaticBindingGenerator()) {
+				let npmConfig = {
+					"save": true,
+					"save-dev": true,
+					"save-exact": true,
+					"silent": true
+				};
+
+				_.each(AndroidProjectService.REQUIRED_DEV_DEPENDENCIES, (dependency: any) =>
+					this.$npm.install(`${dependency.name}@${dependency.version}`, this.$projectData.projectDir, npmConfig).wait()
+				);
+			} else {
+				this.$logger.printMarkdown(` As you are using Node.js \`${this.sysInfoData.nodeVer}\` Static Binding Generator will be turned off.` +
```

Comment:
Static Binding Generator will be turned off because you are using Node.js \`${this.sysInfoData.nodeVer}\` .

To use the Static Binding Generator, upgrade your Node.js to ${AndroidProjectService.MIN_REQUIRED_NODEJS_VERSION_FOR_STATIC_BINDINGS} or later.